app/utils.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+from notifications_utils.clients.statsd.statsd_client import StatsdClient
+
+from app.config import config
+
+
+class _StatsdWrapper():
```

Comment:
Maybe call this MetricsClient or have an Abstract class which allows multiple implementation one being a statsd implementation - only thinking that prometheus is around the corner and to minimise any changes.

Reply:
I agree with the renaming, but I think creating an abstract interface to support both statsd's push method and prometheus' pull method needs to be a separate task on its own.